src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -13,12 +13,96 @@
 
 (def master-service-status-version 1)
 
+;; Default list of allowed histograms/timers
+(def default-metrics-allowed-hists
+  ["compiler"
+   "compiler.compile"
+   "compiler.find_facts"
+   "compiler.find_node"
+   "compiler.static_compile"
+   "compiler.static_compile_inlining"
+   "compiler.static_compile_postprocessing"
+   "functions"
+   "http.active-histo"
+   "http.puppet-v3-catalog-/*/-requests"
+   "http.puppet-v3-environment-/*/-requests"
+   "http.puppet-v3-environment_classes-/*/-requests"
+   "http.puppet-v3-environments-requests"
+   "http.puppet-v3-file_bucket_file-/*/-requests"
+   "http.puppet-v3-file_content-/*/-requests"
+   "http.puppet-v3-file_metadata-/*/-requests"
+   "http.puppet-v3-file_metadatas-/*/-requests"
+   "http.puppet-v3-node-/*/-requests"
+   "http.puppet-v3-report-/*/-requests"
+   "http.puppet-v3-static_file_content-/*/-requests"
+   "jruby.borrow-timer"
+   "jruby.free-jrubies-histo"
+   "jruby.lock-held-timer"
+   "jruby.lock-wait-timer"
+   "jruby.requested-jrubies-histo"
+   "jruby.wait-timer"
+   "puppetdb.catalog.save"
+   "puppetdb.command.submit"
+   "puppetdb.facts.find"
+   "puppetdb.facts.search"
+   "puppetdb.report.process"
+   "puppetdb.resource.search"])
+
+;; Default list of allowed values/counts
+(def default-metrics-allowed-vals
+  ["http.active-requests"
+   "http.puppet-v3-catalog-/*/-percentage"
+   "http.puppet-v3-environment-/*/-percentage"
+   "http.puppet-v3-environment_classes-/*/-percentage"
+   "http.puppet-v3-environments-percentage"
+   "http.puppet-v3-file_bucket_file-/*/-percentage"
+   "http.puppet-v3-file_content-/*/-percentage"
+   "http.puppet-v3-file_metadata-/*/-percentage"
+   "http.puppet-v3-file_metadatas-/*/-percentage"
+   "http.puppet-v3-node-/*/-percentage"
+   "http.puppet-v3-report-/*/-percentage"
+   "http.puppet-v3-resource_type-/*/-percentage"
+   "http.puppet-v3-resource_types-/*/-percentage"
+   "http.puppet-v3-static_file_content-/*/-percentage"
+   "http.puppet-v3-status-/*/-percentage"
+   "http.total-requests"
+   "jruby.borrow-count"
+   "jruby.borrow-retry-count"
+   "jruby.borrow-timeout-count"
+   "jruby.num-free-jrubies"
+   "jruby.num-jrubies"
+   "jruby.request-count"
+   "jruby.return-count"
+   "num-cpus"])
+
+; List of allowed jvm gauges/values
+(def default-jvm-metrics-allowed
```

Comment:
Note that the actual code which registers these metrics wouldn't come into OSS Puppet Server code until we do [SERVER-1752](https://tickets.puppetlabs.com/browse/SERVER-1752).  May be best in terms of version history to have this block come in with that ticket but not a big deal either way.